client/my-sites/store/components/action-header/index.js

```diff
@@ -1,62 +0,0 @@
-import { Button, Gridicon } from '@automattic/components';
-import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import SiteIcon from 'calypso/blocks/site-icon';
-import { getSelectedSiteWithFallback } from 'calypso/state/sites/selectors';
-import { setLayoutFocus } from 'calypso/state/ui/layout-focus/actions';
```

Comment:
Another `setLayoutFocus` down, nice 👏

Reply:
Can you tell me more about this :)

Reply:
It's a remnant of the old Calypso! 

It basically allows switching focus between:
- sidebar
- content
- site selector (the old one above the sidebar on the left)
- any other custom layout areas we registered that appear only in certain screens (like `preview`).

While we're not actively removing these, they make little sense now that we don't really use the site switcher (it's actually used in Jetpack Cloud still, see https://github.com/Automattic/wp-calypso/pull/101844 that aimed to remove it but never got to that). Once the new hosting dashboard is more broadly adopted, they'll be completely obsolete.